components/button/Button.styles.js

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+const BlueButton = styled.button`
+  color: white;
+  background-color: rgb(0, 121, 211);
+  :hover {
+    background-color: #7fbce9;
+  }
+  border-radius: 4px;
+  font-size: 12px;
+  width: 106px;
+  height: 32px;
+  border: 1px solid transparent;
+  font-weight: 700;
+  font-family: 'IBM Plex Sans', sans-serif;
```

Comment:
This needs to be removed. Global css should enforce font family for everything in the app.

Reply:
Done!